src/NHSDPublicBrowseAcceptanceTests.Tests/Steps/Authorization/Authorization.cs

```diff
@@ -0,0 +1,118 @@
+﻿using FluentAssertions;
+using NHSDPublicBrowseAcceptanceTests.Tests.Utils;
+using OpenQA.Selenium;
+using System;
+using TechTalk.SpecFlow;
+
+namespace NHSDPublicBrowseAcceptanceTests.Tests.Steps.Authorization
+{
+    [Binding]
+    public class Authorization
+    {
+        private readonly ScenarioContext _context;
+        private readonly UITest _test;
+
+        public Authorization(UITest test, ScenarioContext context)
+        {
+            _test = test;
+            _context = context;
+        }
+        
+        [Given(@"that a User is not logged in")]
+        public void GivenThatAUserIsNotLoggedIn()
+        {
+            var builder = new UriBuilder(_test.Driver.Url)
```

Comment:
should this bit be moved out somewhere, can called like OpenLoginPage or something?
Also what happens when we're not running on windows?

Reply:
This is getting removed altogether in the next commit